### DISCUSSION TITLE
Conn error

### DISCUSSION THREAD

**Discussion by HeavySnowJakarta** [blinksh/blink#1713] [Support]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

**iPad layer**
Blink version: 16.0.4.644
iPadOS version: 15.5
iSH version: 1.2.3, build 298
**Alpine layer (as SSH server)**
Alpine version: v3.14 i686
Kernel: 4.20.69-iSH
OpenSSH-8.6p1, OpenSSL 1.1.1l

### Describe the bug

When I link to iSH (of the same device) by SSH using Blink, it outputs:
```
Error connecting to self. connError(msg: "Socket error: No such file or directory")
```
But using clients like Termius or iSH itself works.

**Comment by adamcstephens** [comment]
I received this error when my host wasn't configure to use a key. Once I set the host to have a key (from secure enclave in my case), I was able to connect.

**Comment by carloscabanero** [comment]
Check out what @adamcstephens mentioned in case it helps. What this error means though, is that it could not resolve the address. "No file or directory" is the generic description for that error message, as opening a socket at this level is like opening a file. Make sure you have configured the host properly, things like the name, ports, etc...
